components/card/card.html

```diff
@@ -0,0 +1,27 @@
+{# @card
```

Comment:
Few things about this component:
- The pattern of displaying the page seems a bit odd here. When I click `Card` from the sidemenu, it takes me to `http://localhost:3000/components/detail/card`, which I then can't access back if I click on one of the variants, and I'd have to click on `Card` again to see this page. Seems like an anti-pattern to what we've been structuring so far? I would have assumed this page, that showcases all the different cards, to probably be an `All cards example` page or something, similar to [what you did here](https://demo1--mozilla-protocol.netlify.app/components/detail/example-form.html). What do you think?
- The hierarchy of displaying the variants also could be in an ascending or descending order (like we have in https://protocol.mozilla.org/patterns/molecules/card.html that displays the Large card first, and descends to the Extra Small card), as it would be easier to navigate? I think you chose Small to be the default/first item in the list because it doesn't need an extra sizing class, but I think it's still better to have a clearer hierarchy on the list?
- I also see you added the `preview-dark.html` file in this PR to probably show a dark variant of the card. It would be nice to also include that if you could?

Reply:
Yeah, this got a little weird... Small is the default, but I wanted some separate "overview" documentation not attached to a variant. So I used the collated: true setting which tells Fractal to put all the variants on one page... but then you can't easily get back to that overview. So I'm not sure I like collated components and maybe we should avoid that.

I've updated to add a "Default" and treat "Small" as a variant, even if they're more or less the same thing, but that it least still keeps things orderly.

I think we tend to list sizes from small to large so I was just trying to be consistent. If we prefer listing from large to small we should do that everywhere. I'm not sure I have a strong opinion on which, but we should pick one pattern and stick to it.

The dark preview already existed, I just made a minor edit to it in this PR. I did have a dark card at one point but decided to remove it.

Reply:
Happy to stick to ascending order (small to large).
I do however think you should either add a variant to show the dark card, or to at least include in the notes section that you can add an `mzp-t-dark` class for cards